README.md

```diff
@@ -1,2 +1,110 @@
 # api.webmaker.org
-Services for Webmaker
+
+This is where we'll build the service for saving, publishing, and searching projects created by the Webmaker Android application.
```

Comment:
Isn't the long-term goal to have more than just the android app consume this service?